src/main/java/org/sonarlint/intellij/ui/nodes/LiveSecurityHotspotNode.java

```diff
@@ -68,6 +78,19 @@ public void render(TreeCellRenderer renderer) {
     }
   }
 
+  @NotNull
+  private static Icon[] getIcons(Icon typeIcon, Optional<ServerConnection> serverConnection) {
```

Comment:
Code smell: avoid usage of optional as paremeter

Reply:
Fixed